pages/querying/vector-search.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Vector search
+description: Learn how to use vector search and manage vector indices in Memgraph.
+---
+
+import { Callout } from 'nextra/components'
+
+# Vector search
+
+Vector search, also known as vector similarity search or nearest neighbor search, is a technique used to find the most similar items in a collection of data based on their vector representations.
+The vector search is currently an [experimental
+feature](/database-management/experimental-features). 
+Memgraph implements a READ_UNCOMMITTED isolation level specifically for vector indices. While the main database can operate at any isolation level, the vector index specifically operates at READ_UNCOMMITTED. 
+This design maintains all transactional guarantees at the database level - only the vector index operations use this relaxed isolation level, ensuring the database's ACID properties remain intact for all other operations.
+
+## Run Memgraph with vector search feature 
+
+To try out vector search feature you need to **enable** it and **configure** it.
+To enable the feature set the `--experimental-enabled` to `vector-search`. To
+configure the feature set the `--experimental-config` flag. 
+
+<Callout type="info">
+Changing the configuration settings depends on the way you are using Memgraph,
+so please refer to the [configuration
+docs](/database-management/configuration#changing-configuration) for more
+information.
+</Callout>
+
+Here is the example configuration:
+
+```shell
+--experimental-config=
+'
+{
+  "vector-search": {
+    "index_name": {
+      "label": "Node",
+      "property": "vector",
+      "dimension": 2,
+      "capacity": 1000,
+      "metric": "cos",
+      "resize_coefficient": 2,
+    }
+  }
+}
+'
```

Comment:
@DavIvek, comma at the end leads to validation error for me:
`ERROR: failed validation of new value '{"vector-search": {"index_name": {"label": "Node","property": "vector","dimension": 2,"capacity": 1000, "metric": "cos","resize_coefficient": 2,}}}' for flag 'experimental_config'`

```suggestion
--experimental-config=
'
{
  "vector-search": {
    "index_name": {
      "label": "Node",
      "property": "vector",
      "dimension": 2,
      "capacity": 1000,
      "metric": "cos",
      "resize_coefficient": 2
    }
  }
}
'
```

Reply:
Discussed, removing comma